pyanaconda/startup_utils.py

```diff
@@ -197,8 +198,10 @@ def setup_logging_from_kickstart(data):
 
     :param data: kickstart data
     """
-    host = data.logging.host
-    port = data.logging.port
+    runtime_proxy = RUNTIME.get_proxy()
+    logging_data = LoggingData.from_structure(runtime_proxy.Logging)
+    host = logging_data.host
+    port = logging_data.port
```

Comment:
I think there is now unused 'data' argument ? But perhaps pylint should catch that ?